### ISSUE TITLE
Timeout option not working

### ISSUE THREAD

**Issue by flaurida** [playwright-community/expect-playwright#16]
Hi there,

Thanks for this library! I am working to integrate it with [qawolf](https://github.com/qawolf/qawolf), and I noticed that it times out very quickly when looking for text that is not on the page. This is true even if I pass a longer timeout in `options`.

As a point of feedback, I was also surprised that the default timeout was only 1 second. I would have expected something longer like 30 seconds, consistent with the [Playwright API](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options).

Here is a reproduction with qawolf. Even though the text is not on the page and I wanted to wait for 30 seconds before failing, the test seems to fail instantly (Error message: `Error: Timeout exceed for element 'not there'`).

Please let me know if I'm doing something wrong or missing something. Also happy to help fix this issue :)

```js
const qawolf = require("qawolf");

let browser;
let page;

beforeAll(async () => {
  browser = await qawolf.launch({ headless: false });
  const context = await browser.newContext();
  await qawolf.register(context);
  page = await context.newPage();
});

afterAll(async () => {
  await browser.close();
});

test("test", async () => {
  await page.goto("http://todomvc.com/examples/react");
  await page.type("input.new-todo", "create test!");
  await page.press("input.new-todo", "Enter");
  await page.click("input.toggle");
  await page.click("button.clear-completed");
  await expect(page).toHaveText("not there", { timeout: 30000 });
});
```

**Comment by mxschmitt** [comment]
Hey :)

The reason for why I've used a smaller timeout than Playwright does internally was, if you use the `not` selectors, it would also wait 30 seconds. And I felt to wait 30 seconds per default to check if a text is not persistent on a page is way too much. But due this case is very rare and mostly people check on normal expects instead of on not expects, I'm open to change this and document it in the docs about that behaviour.

Your example which you've provided shows a current behaviour which seems like a bug.

I'll provide a fix for that soon.

**Comment by jperl** [comment]
FWIW I also think the timeout should be the same as Playwright's for consistency.

**Comment by mxschmitt** [comment]
Should be fixed in `expect-playwright@0.2.2`. Thanks for reporting!
